lib/link_monitor/use_case/check_monitored_links.rb

```diff
@@ -0,0 +1,50 @@
+module LinkMonitor
+  module UseCase
```

Comment:
I'm not quite sure why we have a `UseCase` module here?

Reply:
We've unwrapped the classes from this module